oadp/bookbag/workshop/content/Conclusion.adoc

```diff
@@ -16,6 +16,6 @@
 |===
 |Type |Link
 |Source Code |https://github.com/openshift/oadp-operator
-|Velero Documentation |https://velero.io/docs/v1.7
+|Velero Documentation |https://velero.io/docs/v1.9/
```

Comment:
might as well use 1.11 so we don't forget

Reply:
1.11 will be for https://github.com/migtools/labs/pull/68

Reply:
Already pushed 1.11 to #68